prac_06/guitars.py

```diff
@@ -0,0 +1,33 @@
+from guitar import Guitar
```

Comment:
This file demonstrates client code usage of the Guitar class, showing how to interact with a list of guitars.